src/main/java/frc/robot/commands/shooter/ShooterCommand.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.commands.shooter;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
+
+public class ShooterCommand extends CommandBase
+{
+    private double velocity;
+    
+    public ShooterCommand(double velocity)
+    {
+        addRequirements(RobotContainer.shooterSubsystem);
+        this.velocity = velocity;
+    }
+    
+    @Override
+    public void initialize()
+    {
+        RobotContainer.shooterSubsystem.setPIDReference(velocity);
+    }
+    
+    @Override
+    public boolean isFinished()
+    {
+        return true;
```

Comment:
Return False, and make the command binding in robot container to while held, when the button is let go the while held binding will automatically cancel the command.